<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDContacts</class>
 <widget class="QMainWindow" name="WDContacts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>570</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>570</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>570</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MailTo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lNickName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddContact">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lvContacts">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>301</width>
      <height>381</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTableWidget::item:selected{ background-color: rgb(198, 196, 199);} </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemoveContact">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>选项</string>
    </property>
    <addaction name="acPreference"/>
    <addaction name="acUserSettings"/>
    <addaction name="acHistory"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acPreference">
   <property name="text">
    <string>偏好设置</string>
   </property>
  </action>
  <action name="acUserSettings">
   <property name="text">
    <string>用户设定</string>
   </property>
  </action>
  <action name="acAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="acHistory">
   <property name="text">
    <string>聊天记录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
